django/db/backends/mysql/features.py

```diff
@@ -30,6 +30,7 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     supports_select_intersection = False
     supports_select_difference = False
     supports_slicing_ordering_in_compound = True
+    strindex_matches_case_insensitively = True
```

Comment:
So maybe it should be named something like `has_case_insensitive_string_comparisons`, but really this should be controlled by the database's collation rather than a hardcoded value, right?